tests/unit/checkboxradio/checkboxradio_methods.js

```diff
@@ -0,0 +1,147 @@
+/*
+* checkboxradio_methods.js
+*/
+
+
+(function($) {
+
+module( "Checkboxradio: methods" );
+test( "Checkbox: refresh", function( assert ) {
+	var widget, icon,
+		checkbox = $( "#checkbox-method-refresh" );
+	expect( 11 );
+	checkbox.checkboxradio();
+
+	widget = checkbox.checkboxradio( "widget" );
+	strictEqual( widget.find( ".ui-icon" ).length, 1,
+		"There is initally one icon" );
+	widget.find( ".ui-icon" ).eq( 0 ).remove();
+	checkbox.checkboxradio( "refresh" );
+	icon = widget.find( ".ui-icon" );
+	strictEqual( icon.length, 1,
+		"Icon is recreated on refresh if absent" );
+
+	assert.hasClasses( icon, "ui-icon-blank" );
+	assert.lacksClasses( icon, "ui-icon-check" );
+	assert.lacksClasses( widget, "ui-checkboxradio-checked" );
+
+	checkbox.prop( "checked", true );
+	checkbox.checkboxradio( "refresh" );
+	assert.hasClasses( icon, "ui-icon-check" );
+	assert.lacksClasses( icon, "ui-icon-blank" );
+	assert.hasClasses( widget, "ui-checkboxradio-checked" );
+
+	checkbox.prop( "checked", false );
+	checkbox.checkboxradio( "refresh" );
+	assert.hasClasses( icon, "ui-icon-blank" );
+	assert.lacksClasses( icon, "ui-icon-check" );
+	assert.lacksClasses( widget, "ui-checkboxradio-checked" );
+});
+
+test( "Checkbox: destroy", function(){
+	expect( 1 );
+	domEqual( "#checkbox-method-destroy", function() {
+		$( "#checkbox-method-destroy" ).checkboxradio().checkboxradio( "destroy" );
+	});
+});
+
+test( "Checkbox: disable / enable", function( assert ) {
+	var checkbox = $( "#checkbox-method-disable" ),
+		widget = checkbox.checkboxradio().checkboxradio( "widget" );
+
+	expect( 4 );
+	checkbox.checkboxradio( "disable" );
+	assert.hasClasses( widget, "ui-state-disabled",
+		"label gets ui-state-disabled when disable is called" );
+	strictEqual( checkbox.is( ":disabled" ), true,
+		"checkbox is disabled when disable is called" );
+	checkbox.checkboxradio( "enable" );
```

Comment:
Add blank line above. In general, group [changes state][asserts state] together.